plugconf/plugconf.go

```diff
@@ -519,6 +543,34 @@ func getDependencies(fn *ast.Function) (pathutil.ReposPathList, error) {
 	return deps, parseErr
 }
 
+// shouldLoadUnconditionally checks if the given function only contains the
+// single default 'return 1' statement which means loading unconditionally.
+// If any other statement is found it is considered a condition.
```

Comment:
Please also change this comment and function name (because not only "ShouldLoad" but "ShouldNotLoad")
`detectUnconditionallyLoad` or something?